diff --git a/src-cljs/cljs101/core.cljs b/src-cljs/cljs101/core.cljs
index b815a4c..a75a36e 100644
--- a/src-cljs/cljs101/core.cljs
+++ b/src-cljs/cljs101/core.cljs
@@ -3,7 +3,8 @@
     [cljs101.exercises.ex101-def-forms]
     [cljs101.exercises.ex102-strings]
     [cljs101.exercises.ex103-keywords]
-    [cljs101.exercises.ex105-maps]))
+    [cljs101.exercises.ex105-maps]
+    [cljs101.exercises.ex106-vectors]))
 
 (defn -main []
   (js/console.log "ClojureScript 101: May the parens be ever in your favor")
